sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Files.DataLake client library</AssemblyTitle>
-    <Version>12.0.0-preview.5</Version>
+    <Version>12.0.0-preview.6</Version>
```

Comment:
Please revert this one - it hasn't been released yet and we'll want `.5` when they do.

Reply:
Oops. Right. it hasn't been released. Reverting. Thanks.